automated/linux/cyclictest/parse_results.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
```

Comment:
Would it make sense to put this file (parse_results.py) in some common place where both cyclictest/ and signaltest/ can use it

Reply:
If the format is the same for cyclictest and signaltest I suggest to move it to automated/lib.

Reply:
Not sure if it's worth sharing this small script between both tests. Unfortunately, all the rt-tools have a quirky output format and I don't know if we can reuse it somewhere else.

Anyway, the upstream plan is to add to all of those tools a machinable parseable output format. I hope this would allow to get rid of those script in the long run.